build-tools/scripts/XAVersionInfo.targets

```diff
@@ -40,7 +40,7 @@
   <Target Name="GetXAVersionInfo"
       DependsOnTargets="_GetSubmodulesVersionInfo">
     <GitBlame
-        FileName="Configuration.props"
```

Comment:
A problem with this change is that it'll *reset* our current commit distances!  Once this is merged, instead of `XA 10.4.99.51` (as of 42822e0488185cdf4bca7c0bd05b21ad03dfbd7e; see https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3687020&view=artifacts&type=publishedArtifacts), it'll become `XA 10.4.99.0`.

If we're going to change the file that contains `$(ProductVersion)`, then we need to *also* bump `$(ProductVersion)` to e.g. `10.4.100.0`, so that "new" master builds don't go "backwards".